src/main/java/org/folio/linked/data/model/entity/Resource.java

```diff
@@ -132,13 +133,15 @@ public Set<ResourceTypeEntity> getTypes() {
     return types;
   }
 
-  public Resource addType(@NonNull ResourceTypeEntity type) {
+  public Resource addTypes(@NonNull ResourceTypeEntity type) {
```

Comment:
minor - it was better to leave the name of this method as `addType` right? (as this method accept only one object as input)

Reply:
oh yep, it was refactored by my IDE automatically, let me fix it